db/migrate/20131017150735_create_automatic_exploitation_match_results.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAutomaticExploitationMatchResults < ActiveRecord::Migration
+  def change
+    create_table :automatic_exploitation_match_results do |t|
+      t.integer :match_id
```

Comment:
This is an old migration from Pro (see the 2013 date). Should fixing this be a new migration?